workalendar/europe/austria.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from workalendar.core import WesternCalendar, ChristianMixin
+
+
+class Austria(WesternCalendar, ChristianMixin):
+    "Austria"
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (5,   1, "National Holiday"),  # Staatsfeiertag
```

Comment:
Dunno, could be, I'm not any kind of expert on these things :)
I generally go with with Wikipedia on these things.
[Austria.info](http://www.austria.info/uk/service-facts/practical-information/public-holidays-daylight-savings-time) looks kinda official and the [literal translation of Staatsfeiertag](https://translate.google.com/#de/en/Staatsfeiertag) is National Day.